README.md

```diff
@@ -50,4 +50,8 @@ To access the system use these credentials:
 
 > _Please, remember to change the password_
 
-An open network server is pre-loaded, therefore, some alerts must appear.
\ No newline at end of file
+An open network server is pre-loaded, therefore, some alerts must appear.
+
+
+## Run RoLaGuard using Makefile
```

Comment:
Hice algunos cambios
## Run RoLaGuard using Makefile
To start up the system locally with docker images, simply run _make all_ command. This will clean up the database, pull the develop branch of all modules, and build the images. It'll take a time to finish its execution, so please be patient. See the [Makefile](./Makefile) to see other options.

Reply:
Agree, but i think this phrase's a little repetitive: "**See** the Makefile to **see** other options."

Reply:
Also, I would add "To start up the system locally _with all the modules running inside docker containers_..."